packages/nx/src/command-line/migrate.ts

```diff
@@ -434,6 +443,80 @@ export class Migrator {
     ]).then((a: { shouldApply: boolean }) => a.shouldApply);
   }
 
+  private getPkgVersion(pkg: string): string {
+    return this.getInstalledPackageVersion(
+      pkg,
+      this.installedPkgVersionOverrides
+    );
+  }
+
+  private normalizePackageGroup(
+    packageName: string,
+    targetVersion: string,
+    packageGroup: PackageGroup
+  ): { package: string; version: string }[] {
+    // Support Migrating to older versions of Nx
+    // Use the packageGroup of the latest version of Nx instead of the one from the target version which could be older.
+    if (
+      packageName === '@nrwl/workspace' &&
+      lt(targetVersion, '14.0.0-beta.0')
+    ) {
+      packageGroup = {
+        '@nrwl/workspace': '*',
+        '@nrwl/angular': '*',
+        '@nrwl/cypress': '*',
+        '@nrwl/devkit': '*',
+        '@nrwl/eslint-plugin-nx': '*',
+        '@nrwl/express': '*',
+        '@nrwl/jest': '*',
+        '@nrwl/linter': '*',
+        '@nrwl/nest': '*',
+        '@nrwl/next': '*',
+        '@nrwl/node': '*',
+        '@nrwl/nx-plugin': '*',
+        '@nrwl/react': '*',
+        '@nrwl/storybook': '*',
+        '@nrwl/web': '*',
+        '@nrwl/js': '*',
+        '@nrwl/cli': '*',
+        '@nrwl/nx-cloud': 'latest',
+        '@nrwl/react-native': '*',
+        '@nrwl/detox': '*',
+        '@nrwl/expo': '*',
+      };
+    }
+
+    if (!packageGroup) {
+      return [];
+    }
+
+    if (!Array.isArray(packageGroup)) {
+      return Object.entries(packageGroup).map(([pkg, version]) => {
+        if (this.installedPkgVersionOverrides[packageName] && version === '*') {
```

Comment:
@FrozenPandaz I made the change so it applies to the `packageGroup` of any package, not just `nx`. Please let me know if we want to scope this change to only apply to the `nx` package (and maybe `@nrwl/workspace` for compat reasons).

Reply:
`migrate` should work the same for community plugins that use package group, or Nx's package group. This is good :)

Reply:
yeah, I agree and that's why I implemented it that way, but wanted to be sure :)